rust/cymbal/README.md

```diff
@@ -1,3 +1,19 @@
 # Cymbal, for error tracking
 
 You throw 'em, we catch 'em.
+
+
+### Terms
+We use a lot of terms in this and other error tracking code, with implied meanings. Here are some of them:
+- **Issue**: A group of errors, representing, ideally, one bug.
```

Comment:
I feel like there's something missing here about fingerprints. Is an issue a collection of fingerprints? Or are issues groups of errors, mapped via fingerprints

Reply:
Issues are collections of fingerprints, but I'm trying to make it explicit that 1 raw stack != 1 fingerprint